others.md

```diff
@@ -9,28 +9,47 @@ The Network Access Control Model (NACM) {{RFC8341}} provides the means to
 restrict access for particular users to a pre-configured subset of all
 available protocol operations and content.
 
-None of the readable data nodes in these YANG modules are considered sensitive or
-vulnerable in network environments. The NACM "default-deny-all" extension has
-not been set for any data nodes defined in these module.
-
-None of the writable data nodes in these YANG modules are considered sensitive
-in network environments. The NACM "default-deny-write" extension has not been
-set for any data nodes defined in these modules.
-
-However, there are a number of writable data nodes defined in these YANG
-modules that are considered vulnerable in network environments. Write
-operations to these data nodes without proper protection can have a negative
-effect. These are the data nodes and their vulnerable:
-
-feed-interval:
-  : A malicious client could attempt to set a very low/large value to this
-  node. Setting a very low value could attack the data source. And setting a
-  very large value would lead to maintaining stale data in the ALTO server.
-
-poll-interval:
-  : A malicious client could attempt to set a very low/large value to this
-  node. Setting a very low value could attack the data source. And setting a
-  very large value would lead to maintaining stale data in the ALTO server.
+There are a number of data nodes defined in these two YANG modules that are
+writable/creatable/deletable (i.e., config true, which is the default).
+These data nodes may be considered sensitive or vulnerable in some network environments.
+Write operations (e.g., edit-config) to these data nodes without proper protection
+can have a negative effect on network operations. These are the subtrees and data
+nodes in "ietf-alto" YANG module and their sensitivity/vulnerability:
+
+  /alto:alto/alto:alto-client/alto:server-discovery:
+  : This subtree specifies a set of parameters for an ALTO client to discover ALTO server.
+  Unauthorized access to it could cause intruders to modify the ALTO discovery
+  parameters(e.g., dns-server) so as to expose an ALTO client to fake ALTO servers.
+
+  /alto:alto/alto:alto-server/logging-system:
+  : This subtree provides configuration to select a logging system to capture log
+  messages generated by an ALTO server. Unauthorized read access of this node
+  can allow intruders to access logging information, which could be used to craft
```

Comment:
This item is about readable concerns, while the items in this list should focus on "writable/creatable/deletable". Please move this one to be positioned right after: 

> Some of the readable data nodes in this YANG module may be considered sensitive or vulnerable in some network environments. It is thus important to control read access (e.g., via get, get-config, or notification) to these data nodes. These are the subtrees and data nodes and their sensitivity/vulnerability:

Reply:
Sure, finished.